CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1 @@
+All	participants agree to abide by LF Projects Code of Conduct (as defined in the [charter](tsc/charter.md)) available at https://lfprojects.org/policies/code-of-conduct/
```

Comment:
Some funky white-space here in the first couple of words?

Reply:
Yes - good catch. Fixed now.